ChangeLog.md

```diff
@@ -26,7 +26,6 @@ Current Trunk
 - Removed obsolete SIMD.js support (-s SIMD=1). Use -msimd128 to target Wasm
   SIMD. (#11180)
 - Add warning about fastcomp deprecation (can be disabled via `-Wno-fastcomp`).
-- Removed obsolete SIMD.js support (-s SIMD=1). Use -msimd128 to target Wasm SIMD. (#11180)
```

Comment:
Why remove this line? I think this is still relevant.

Reply:
It's a removal of a duplicate. See 2 lines above.